pyproject.toml

```diff
@@ -82,7 +82,7 @@ exporters = [
     "factorio-randovania-mod>=0.5.0",
 
     # Metroid Dread
-    "open-dread-rando>=2.15.0",
+    "open-dread-rando>=2.16.0",
```

Comment:
That's hard required for the changes.
Add it to the `requirements.txt`

Reply:
Well I did, but apparently per #7740 it's already in main. It's part of my commit but it isn't showing up in this PR because of that, lol

Reply:
Oops, my fault. Haha, I merged that. Now I remember that my thought was "it only adds a (currently) unused feature and I can just merge it without a changelog"